share/lib/python/scripts/_binwrapper.py

```diff
@@ -40,6 +40,26 @@ def _set_default_compiler():
     os.environ.setdefault("CXX", ccompiler.compiler_cxx[0])
 
 
+def _check_cpp_compiler_version():
```

Comment:
We don't seem to use this anywhere, so maybe remove it?

Reply:
in master, this is use in this file in the `main` function.

Add and use in this patch: https://github.com/neuronsimulator/nrn/commit/252d88ddee43ebc187d722edf7c25bbe2facd3c1

Reply:
Can I just add this line here: https://github.com/neuronsimulator/nrn/commit/252d88ddee43ebc187d722edf7c25bbe2facd3c1#diff-915d6b5c07819099da5fde35dc66535dfdee67280f086c81b3540bbaff8ce95fR115
or should I add more stuff from that commit?